suite/auto-sync/CppTranslator/CppTranslator.py

```diff
@@ -406,6 +406,13 @@ def run_clang_format(self):
             log.info(f"Format {out_file}")
             subprocess.run(["clang-format-13", "-style=file", "-i", out_file])
 
+    def remark_manual_files(self) -> None:
+        manual_edited = self.conf["manually_edited_files"]
+        if len(manual_edited) > 0:
+            log.warning("The following files are to complex to translate! Please check them by hand.")
```

Comment:
`too complex`

Reply:
It probably also makes sense to make this warning more visible to the user instead of a plain text message.